lib/chainsync/chainsync/test_fixtures/db_session.py

```diff
@@ -26,7 +27,14 @@ def psql_docker() -> Iterator[PostgresConfig]:
     Iterator[PostgresConfig]
         An iterator that yields a PostgresConfig
     """
-    client = docker.from_env()
+    home_dir = os.path.expanduser("~")
+    socket_path = Path(f"{home_dir}/.docker/desktop/docker.sock")
```

Comment:
is this an OS independent path? I think you want to use [os.path.join](https://docs.python.org/3/library/os.path.html#os.path.join) instead of `/`

Reply:
it seems you're right, hardcoding forward slash is bad
Path is the more modern library and their `/` operator chooses the correct type of slash based on your OS
gonna use `socket_path = Path(home_dir) / ".docker" / "desktop" / "docker.sock"`